plugins/signals-backend/src/service/SignalManager.ts

```diff
@@ -47,18 +46,19 @@ export class SignalManager {
     string,
     SignalConnection
   >();
-  private eventBroker?: EventBroker;
+  private events: EventsService;
   private logger: LoggerService;
 
   static create(options: SignalManagerOptions) {
     return new SignalManager(options);
   }
 
   private constructor(options: SignalManagerOptions) {
-    ({ eventBroker: this.eventBroker, logger: this.logger } = options);
+    ({ events: this.events, logger: this.logger } = options);
 
-    this.eventBroker?.subscribe({
-      supportsEventTopics: () => ['signals'],
+    this.events.subscribe({
+      id: 'signals',
+      topics: ['signals'],
       onEvent: (params: EventParams<SignalPayload>) =>
```

Comment:
Yep, these were typed in the previous `EventBroker` so that's why added those. Not sure if the new `EventsService` should be made to (optionally) support `zod` schemas for the events in both publishing and subscribing?

Reply:
I will remove this typing support for now, maybe discuss later for possible `zod` support for the events?

Reply:
sounds good! 👍